### DISCUSSION TITLE
Does rye expose the `uv` installation that it uses?

### DISCUSSION THREAD

**Discussion by nikhilweee** [astral-sh/rye#1277] [Q&A]
I have configured rye to use the uv under the hood. I see that there's a `uv` folder located inside `~/.rye`. Is there a way to use this installation of uv? Is it even recommended to do so? The reason I ask is that if uv comes preinstalled with rye, it would save the user from installing uv separately. 

My particular use case is that I want some packages in my venv but not in `pyproject.toml`, so I do a `uv pip install <package>` instead of `rye add <package>`. Right now I have to install uv separately if I intend to do so.

**Comment by bluss** [comment]
It sounds like a use case for dev-dependencies, is that something you could use?

It seems like formally rye does not expose its uv installation. You could install it separately using "rye tools install".
